tools/releases/sdf-release.yml

```diff
@@ -0,0 +1,80 @@
+trigger: none
+resources:
+  repositories:
+    - repository: GovernedTemplates
+      type: git
+      name: OneBranch.Pipelines/GovernedTemplates
+      ref: refs/heads/main
+  pipelines:
+    - pipeline: _OfficeDev.microsoft-teams-library-js
+      source: OfficeDev.microsoft-teams-library-js
+      project: ISS
+variables:
+  - name: System.Debug
+    value: true
+extends:
+  template: v2/Microsoft.Official.yml@GovernedTemplates
+  parameters:
+    platform:
+      name: m365
+      workload: IC3
+      serviceTreeId: $(serviceTreeId)
+      serviceGroupName: Platform
+    stages:
+      - stage: Prod_Lockbox_Approval_Deployment
+        displayName: Lockbox Approval/Deployment
+        dependsOn: []
+        variables:
+          stage_type: deployment
+          azure_subscription_ids: $(subscriptionId)
+        jobs:
+          - job: Agent_job
+            pool:
+              type: release
+            steps:
+              - download: _OfficeDev.microsoft-teams-library-js
+              - task: prepare-deployment@1
+                displayName:
+                inputs:
+                  taskType: credentialFetchTaskAzureRM
+                  armserviceconnection: $(serviceConnectionId)
+                  subscriptionid: $(subscriptionId)
+              - task: AssetRetention@3
+                displayName: ARtifact Retention Orchestrator Workflow (ARROW)
```

Comment:
>ARtifact [](http://example.com/codeflow?start=29&length=8)

Artifact

Reply:
Strangely enough this is actually on purpose to hit the ARROW acronym. Otherwise it would've been Arow (not controlled by our team).

Reply:
lol